tracecat/expressions/functions.py

```diff
@@ -98,6 +98,46 @@ def format_string(template: str, *values: Any) -> str:
     return template.format(*values)
 
 
+def capitalize(x: str) -> str:
+    """Capitalize a string."""
+    return x.capitalize()
+
+
+def titlize(x: str) -> str:
```

Comment:
nit: either [`titleize`](https://www.google.com/search?sca_esv=7025c20cef8c26a5&q=titleize&spell=1&sa=X&ved=2ahUKEwi8sfbM7u6JAxVCFjQIHZNwDeYQBSgAegQIFhAB&biw=1554&bih=1019&dpr=2) (google autocorrects titlize to this) or `to_title` or just `title` (like how `capitalize` is just capitalize)

Reply:
```suggestion
def titleize(x: str) -> str:
```